README.md

```diff
@@ -1,2 +1,201 @@
 # Capstone-Project
 AniverseStudio is a site for anime nerds. We showcase our latest features, news and events etc.
+# Portfolio 
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <h3><b>Portfolio Project Readme</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Portfolio-Project] <a name="about-project"></a>
+
+
+**[Portfolio-Project]** is an initiative aimed at developing proficiency in Figma-based user interface (UI) designs and building a personalized portfolio website. This project endeavors to showcase optimal design abilities and web development skills.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">Html</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Responsive Design]**
+In this project, we have implemented responsive design to ensure that our application looks and functions correctly across various devices and screen sizes. This project uses a Mobile-first approach to ensure that it works responsive with smaller screen and scale up to meet the need of larger screens.
+We have achieved this through the use of media queries and flexible layouts, which adapt to the user's device and provide them with an optimal viewing experience. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
+
+#### You can visit the live website from here
+- [Live Demo Link](https://MoyasiGinko.github.io/Portfolio-Site/)
```

Comment:
- [ ] Also i would kindly suggest you add the correct deployed link to the live demo section, what we currently have here is the deployed link of the portfolio project.
- [ ] Also, as part of the requirement to this project, we are asked to make a loom recording of 5minute max. where we showcase the project and as well show some piece of code we have implemented that we are most proud of. Kindly record the loom video and add the link in your readme file.